.nancy-ignore

```diff
@@ -0,0 +1,5 @@
+# Since nancy doesn't respect replace directive, this check will fail until
+# upstream tendermint depends on websocket@1.4.0. We force a fixed version of
+# the websocket in go.mod. Remove once upstream dependencies do not depend on
+# websocket@1.4.0 and replace directive is not needed anymore.
+5f259e63-3efb-4c47-b593-d175dca716b0
```

Comment:
What's this magic UUID?

Reply:
https://ossindex.sonatype.org/vuln/5f259e63-3efb-4c47-b593-d175dca716b0

I'll add a comment with the link above it. Also tests passed and this is only change that i'll do, so i'll just merge it without waiting for the tests again

Reply:
Yeah go go go :+1: